internal/jimm/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2025 Canonical.
+
+package bootstrap
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/errors"
+	"github.com/canonical/jimm/v3/internal/openfga"
+	"github.com/canonical/jimm/v3/pkg/api/params"
+)
+
+// JobTracker interface defines the methods required for job tracking.
+type JobTracker interface {
+	// GetJob retrieves a job entry by its ID.
+	GetJob(ctx context.Context, jobId uuid.UUID) (dbmodel.JobTrackerEntry, error)
+}
+
+type bootstrapManager struct {
+	jobtracker JobTracker
+	store      *db.Database
```

Comment:
Why is there a job tracker and the store? Is that because the logs need the DB but you want to test with mock jobs?